src/main/java/skhu/hanziboong/schedule/controller/ScheduleController.java

```diff
@@ -0,0 +1,25 @@
+package skhu.hanziboong.schedule.controller;
+
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import skhu.hanziboong.schedule.dto.request.ScheduleRequest;
+import skhu.hanziboong.schedule.service.ScheduleService;
+
+@RestController
+@RequiredArgsConstructor(access = AccessLevel.PROTECTED)
+@RequestMapping("/api/schedule")
+public class ScheduleController {
+
+    private final ScheduleService scheduleService;
+
+    @PostMapping
+    public ResponseEntity<Void> createSchedule(@RequestBody ScheduleRequest request) {
+        scheduleService.createSchedule(request);
+        return ResponseEntity.ok().build();
```

Comment:
```suggestion
        return ResponseEntity.created(URI.create("/api/schedule/" + response.id())).build();
```

좀 더 구체적인 HTTP 상태 코드를 반환하는 편이 좋을 것 같은데 어떻게 생각하시나요?

Reply:
`ApiResponse`를 제작해두었는데 리뷰를 받고나서 도입하고자 해서 이렇게 작성했던 것 같아요
저런식으로 http 상태 코드를 반환하는 방법에 대해서는 잘 몰랐던 것 같아요
배워갈 수 있었습니다! ㅎㅎ